app/models/planning_application.rb

```diff
@@ -350,11 +350,25 @@ def documents_for_decision_notice
     documents.for_display
   end
 
+  def received_at
```

Comment:
we also have a helper method with this name in planning application helpers, should we get rid of the helper and replace view calls to received_at with the calls to the model method?

Reply:
Yes good spot, thanks :eyeglasses: